.azure-pipelines.yml

```diff
@@ -52,5 +53,13 @@ resources:
       image: ccfciteam/ccf-ci-18.04-oe-0.8.0-rc1:latest
       options: --publish-all --device /dev/sgx:/dev/sgx -v /dev/shm:/ccache
 
+variables:
+  ${{ if startsWith(variables['Build.SourceBranch'], 'refs/tags/v') }}:
```

Comment:
I am not 100% happy with this bit. I think we can revisit this in a future PR.

Reply:
Agreed but hard to imagine how it could be different? There'd need to be a way to... tag the tags so we know they're meant for release?